### PR TITLE
Clarifications and typos

### PR DESCRIPTION
**Pull request by Frenchris**
Small improvement:
Grammatical clarifications.
Adding of some examples in some cases to match the tests.

### CODE REVIEW

File: subjects/wdmatch/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Instructions
 
-Write a program that takes two `string` and checks whether it is possible to write the first `string` with characters from the second `string`, while respecting the order in which these characters appear in the second `string`.
+Write a program that takes two `string` and checks whether it is possible to write the first `string` with characters from the second `string`. This rewrite must respectSS the order in which these characters appear in the second `string`.
```
Comment (xpetit) [inline subjects/wdmatch/README.md:5]: "respectSS" ? 🤔

**Review by xpetit — CHANGES_REQUESTED** [review]

**Comment by xpetit** [comment]
Nice

File: subjects/wdmatch/README.md (hunk 1 above)
Comment (Frenchris) [reply subjects/wdmatch/README.md:5]: woops

Comment (Frenchris) [reply subjects/wdmatch/README.md:5]: ça c est un control+s mal executé héhéhé

Comment (xpetit) [reply subjects/wdmatch/README.md]: > Les gagnants trouvent des moyens, les perdants des excuses

**Pull request merged**